fibonacci.asm

```diff
@@ -18,19 +18,34 @@
 
 fibs:	.word   0 : 9         # create an array variable named "fibs" of 9 word-length elements (4 bytes each)
 size:	.word  9              # create a single integer variable named "size" that indicates the length of the array
+query:	.asciiz "How many fibonacci numbers do you want to calculate (select up to between 2 and 9):\n"
 
 ###############################################
 # .text segment. Assembly instructions go here.
 ###############################################
 	.text
-	      la   $s0, fibs        # load address of array into $s0
+	
+userInput:    la  $a0, query
+	      li  $v0, 4
+	      syscall
+	      
+	      li $v0,5
+	      syscall
+	      la $s6, ($v0)
+	      
```

Comment:
-5 Accepts input greater than 9.  This allows storing numbers beyond the bounds of the array.